packages/jest-diff/README.md

```diff
@@ -0,0 +1,238 @@
+# jest-diff
+
+Display differences clearly so people can review changes confidently.
+
+The default export serializes JavaScript **values** and compares them line-by-line.
+
+Two named exports compare **strings** character-by-character:
+
+- `diffStringsAligned` returns a string which includes comparison lines.
+- `diffStringsUnaligned` returns an array of 2 strings.
+
+## Installation
+
+To add this package as a dependency of a project, run either of the following commands:
+
+- `npm install jest-diff`
+- `yarn add jest-diff`
+
+## Usage of default export
+
+Given values and optional options, `diffLines(a, b, options?)` does the following:
+
+- **serialize** the values as strings using the `pretty-format` package
+- **compare** the strings line-by-line using the `diff-sequences` package
+- **format** the changed or common lines using the `chalk` package
+
+To use `diffLines` as the function name, write either of the following:
+
+- `const diffLines = require('jest-diff');` in a CommonJS module
```

Comment:
we'll probably go for named exports in jest 25 (we wanna avoid having to use `export =`), but this is of course fine for now